Doc/library/io.rst

```diff
@@ -833,8 +833,7 @@ than raw I/O does.
 .. class:: BufferedRandom(raw, buffer_size=DEFAULT_BUFFER_SIZE)
 
    A buffered binary stream providing higher-level access to a seekable
-   :class:`RawIOBase` raw binary stream.  It inherits from :class:`BufferedReader`
-   and :class:`BufferedWriter`.
+   :class:`RawIOBase` raw binary stream.
```

Comment:
I think this looks good as is. Mainly exploring some. What about something like:

```suggestion
   A buffered binary stream implementing :class:`BufferedIOBase` interfaces
   providing higher-level access to a seekable :class:`RawIOBase` raw binary 
   stream.
```